jina/types/sets/document.py

```diff
@@ -63,6 +63,19 @@ def __getitem__(self, item):
         else:
             raise IndexError(f'do not support this index {item}')
 
+    def __add__(self, other: 'DocumentSet'):
+        sum = DocumentSet([])
```

Comment:
sum seems like a weird name?

Reply:
shadowing built-in, better use other, e.g. `_tmp`, or simply `v`

Reply:
thanks changed